layerInfo.json

```diff
@@ -43,7 +43,7 @@
 			},
 			"Driver-Utils": {
 				"packages": ["@fluidframework/driver-utils"],
-				"deps": ["Protocol-Utils", "Driver-Definitions", "Telemetry-Utils"]
+				"deps": ["Protocol-Utils", "Driver-Definitions", "Telemetry-Utils", "Core-Utils"]
```

Comment:
Are these safe changes to make? I think it's ok since Core-Utils has no dependencies.

Reply:
From the core-utils README, it sounds like the new-and-improved common-utils just for the client release group, so makes perfect sense to me